constellation_ticketbox/views.py

```diff
@@ -320,17 +328,18 @@ def api_v1_ticket_update_status(request, ticket_id):
         box = ticket.box
         box_perms = get_perms(request.user, box)
 
-        if ((ticket.owner == request.user) or ('action_manage_tickets' in box_perms)):
+        if ((ticket.owner == request.user) or
```

Comment:
For some reason using `is` will cause this test to always fail, but it has the correct functionality using `==`.